t/plugin/consumer-restriction.t

```diff
@@ -263,7 +263,8 @@ Authorization: Basic amFjazIwMjA6MTIzNDU2
                             "consumer-restriction": {
                                  "blacklist": [
                                      "jack1"
-                                 ]
+                                 ],
+                                 "rejected_message": "request is forbidden"
```

Comment:
New test cases are needed to ensure that the new feat works as expected.

Reply:
This change is already a new case. But we need to make sure it can pass.

Reply:
@qq54903099 
Please update the test to make CI pass: https://github.com/apache/apisix/runs/4465467353?check_suite_focus=true